docs/src/components/mdx/dynamic-code-block-overlay.tsx

```
@@ -0,0 +1,273 @@
+
+        {/* Content */}
+        <div className="overflow-auto">
+          <div
+            className="p-3 sm:p-4"
+            style={{
+              background: '#0a0a0a'
+            }}
+          >
+            <div
+              className="[&_*]:!bg-transparent [&_pre]:!bg-transparent [&_code]:!bg-transparent text-xs sm:text-sm leading-[1.4] sm:leading-[1.5] [&_pre]:text-xs [&_pre]:sm:text-sm [&_code]:text-xs [&_code]:sm:text-sm [&_pre]:leading-[1.4] [&_pre]:sm:leading-[1.5] [&_code]:leading-[1.4] [&_code]:sm:leading-[1.5] [&_pre]:m-0 [&_pre]:p-0 [&_pre]:overflow-visible"
+              dangerouslySetInnerHTML={{ __html: highlightedCode }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)